tests/test_bench.py

```diff
@@ -35,13 +44,13 @@
 from pytket.extensions.qiskit import tk_to_qiskit
 from qiskit import QuantumCircuit
 
-test_qasm_output_path = "./test_output/"
-
 
-def test_configure_begin():
-    Path(test_qasm_output_path).mkdir(parents=True, exist_ok=True)
-    utils.set_qasm_output_path(test_qasm_output_path)
-    assert utils.get_qasm_output_path() == test_qasm_output_path
+@pytest.fixture()
+def output_path():
+    output_path = Path("./tests/test_output/")
+    if not output_path.exists():
+        output_path.mkdir()
```

Comment:
There was a one line syntax for that. Can't quite remember at the top of my head.

Reply:
Fixed that.